akka-cluster/src/multi-jvm/scala/akka/cluster/StressSpec.scala

```diff
@@ -126,11 +129,6 @@ private[cluster] object StressMultiJvmSpec extends MultiNodeConfig {
     akka.loggers = ["akka.testkit.TestEventListener"]
     akka.loglevel = INFO
     akka.remote.log-remote-lifecycle-events = off
-
-    akka.remote.artery.advanced.aeron {
-      idle-cpu-level = 1
-    }
-
     akka.actor.default-dispatcher.fork-join-executor {
       parallelism-min = 8
       parallelism-max = 8
```

Comment:
this can probably be removed now, it was probably there to reduce number of threads but now we have a lower factor